front/src/components/App/SalesArea/SalesArea.tsx

```diff
@@ -2,37 +2,85 @@ import Button from "../../UI/Button/Button";
 import Header from "../../UI/Header/Header";
 import RouteLink from "../../UI/RouteLink/RouteLink.styled";
 import { APP_PAGE } from "../App";
-import allegro_image from "../../Images/allegro.jpg";
-import olx_image from "../../Images/olx.jpg";
-import ebay_image from "../../Images/ebay.jpg";
-import CompaniesSector from "../../UI/CompanyView/CompaniesSector";
+import sony_image from "../../Images/sony.jpg";
+import jbl_image from "../../Images/jbl.jpg";
+import steelseries_image from "../../Images/myszka.jpg";
+import Products from "../../UI/ProductView/ProductSector";
 import { SalesAreaStyled } from "./SalesArea.styled";
+import Auctions from "../../UI/AuctionView/AuctionSector";
+import React, { useState } from "react";
+import ProductFilter from "../../UI/ProductView/ProductFilter";
 
-const EXAMPLES_COMPANIES = [
+enum InitState {
+  name = "Products",
```

Comment:
elementy w enumie raczej nazywa się WIELKIMI_LITERAMI. Z tym że ten enum to nie InitState tylko coś co opisze czym jest to value w dalszej części pliku.
```
enum AdekwatwaNazwa {
  PRODUCTS = "products",
  ...
}
```